### PR TITLE
Feature/schema update

### LINKED ISSUE #72: Update db schema
We need two new tables one for Subjects and one for Tags.  The subject and tag ids should replace the values currently on the Quiz table for each respective column (subjects and tags).

### PR DESCRIPTION
**Pull request by ghost**
PR includes:

Updating the schema.

updated_schema

New npm scritps 
- "reset-db"
  - This script removes the docker "volume" which persists data between container restarts.  It is necessary when updating the schema to remove this volume. I got tired of manually typing in the commands necessary for this so extracted it into an npm script.  In the future I may move this to it's own DB scripts folder as I feel conflicted about it sitting top level in the project but it's a good script for the rest of the team to have at the moment.
- "update-schema"
  - This script dumps the local database's schema into the quiz_db.sql.  Running this command will overwrite the current schema found in quiz_db.sql so only use it when you're ready to push a new schema for the rest of the team to use as this file is automatically picked up in the docker initialization for configuring the local database.  Again may move this to a DB scripts folder in the future but may be good for other members of the team to have this script as with the above script, these are a pain to have to type out manually every time.

### CODE REVIEW

**Review by rmcguire6 — APPROVED** [review]
Works well. Thanks.

File: package.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
     "dev-setup": "npm run root-setup && npm run client-setup && npm run dev",
     "client-dev": "cd client && npm run dev",
     "root-setup": "npm ci",
-    "client-setup": "cd client && npm ci"
+    "client-setup": "cd client && npm ci",
+    "reset-db": "docker container rm moonshot-chingu-quiz_db_1 && docker volume rm moonshot-chingu-quiz_quizdata",
+    "update-schema": "docker exec moonshot-chingu-quiz_db_1 pg_dump -U docker docker > quiz_db.sql"
```
[the remark this answers is not in the document]
Comment (rmcguire6) [reply package.json:28]: It's not for the production branch. It's for our local repositories to make working with the DB easier.

[the remark this answers is not in the document]
Comment (rmcguire6) [reply package.json:28]: You have to run in your local repository for anything to happen.

[the remark this answers is not in the document]
Comment (silentkdev) [reply package.json:28]: Yeah we'd be able to see that someone changed the schema during a pull request if they were trying to do that.

**Review by ZumDeWald — APPROVED** [review]

**Pull request merged**